masterlist.yaml

```
@@ -8170,6 +8170,7 @@ plugins:
       - '018Auri.esp'
       - 'FCO - Follower Commentary Overhaul.esp'
       - 'Immersive Citizens - AI Overhaul.esp'
+      - 'alternate start - live another life.esp'
```

Just for consistency I'd change the letter case & order.
```suggestion
      - '018Auri.esp'
      - 'Alternate Start - Live Another Life.esp'
      - 'FCO - Follower Commentary Overhaul.esp'
      - 'Immersive Citizens - AI Overhaul.esp'
```